app/views/home/index.html.haml

```diff
@@ -84,8 +84,28 @@
         %div.align-items
           %h4.margin-items= t('home.fairness')
           = render Display::InfoTooltipComponent.new(text: "You are seing the average scores for all the public ontologies in AgroPortal. FAIR scores are computed with the O'FAIRe methodology. More details here: https://github.com/agroportal/fairness")
+          = link_to(get_fairness_service_url,id: "fairness-service-url", target: "_blank", "aria-label": t("get_json_version"), title: t("get_json_version")) do
```

Comment:
for this one, use the same icon as the one on the summary page. 
![image](https://github.com/ontoportal-lirmm/bioportal_web_ui/assets/29259906/b259df50-b15b-47ca-9642-8d29786643f0)

Same change to the one, on the summary page: (you can extract a helper that your will use in both, homepage and summary if possible)
![image](https://github.com/ontoportal-lirmm/bioportal_web_ui/assets/29259906/448d0ce2-9c1c-4251-844d-5a420ed316ee)

Reply:
okay, i will change it into helper